lib/facter/iis_version.rb

```diff
@@ -11,7 +11,7 @@
                 'powershell.exe'
                end
 
-      iis_ver = %x{#{psexec} -ExecutionPolicy ByPass -Command "(Get-ItemProperty HKLM:\\SOFTWARE\\Microsoft\\InetStp\\ -Name VersionString).VersionString.SubString(8,3)"}
+      iis_ver = %x{#{psexec} -ExecutionPolicy ByPass -Command "(Get-ItemProperty HKLM:\\SOFTWARE\\Microsoft\\InetStp -ErrorAction SilentlyContinue -Name VersionString).Versionstring -replace 'Version ',''"}
```

Comment:
```
FOO:\\BAR\\Baaz\\Zomg
```
reads pretty awful. shouldn't we have a wrapper for that?
i also thought powershell transparently understands (paths also when written as) `FOO:/BAR/Baaz/Zomg` (but maybe that's only true for path paths, not registry paths)